sources/scss/finder.scss

```diff
@@ -0,0 +1,140 @@
+@import"constants";
+.finder{
+	width:100%;
+
+	background-image: url("https://github.com/maxim-maidan/travel-website/blob/finder/img/explore-and-travel.png?raw=true");
+	background-repeat: no-repeat;
+	background-size: auto 90%;
+	background-position:right;
+
+	display:flex;
+	flex-direction:row;
+	padding: 0 0 0 $padding-left;
+
+	.explore{
+		display:flex;	
+		flex-direction:column;
+		h1{
+			width:470px;
+			padding:69px 0 0 0;
+
+			font-family:serif;
+			font-weight:600;
+			font-size: $title;
+			letter-spacing: -0.9px;
+			line-height:1.16;
+			color:$dark-blue;
+
+			@media screen and (max-width:576px){
+				&{
+					font-size: $title-phone;
+					width:100%;
+					padding:69px 50px 0;
+				}
+			}
+		}
+		.input{
+			width:470px;
+			height:auto;
+
+			display:flex;
+			flex-direction:row;
+			justify-content:space-between;
+			flex-wrap:wrap;
+
+			h1{
+				width:100%;
+				padding:72px 0 0 2px;
+
+				font-family: 'Roboto', sans-serif;
+				font-weight:600;
+				font-size: $subtitle;
+				letter-spacing: -0.1px;
+				line-height:1.14;
+				color:$dark-blue;
+
+				@media screen and (max-width:576px){
+					&{
+						font-size: $subtitle-phone;
+						padding:72px 0 0 0;
```

Comment:
if you have only one value in `padding` use `padding-top: 72px` instead of `padding:72px 0 0 0`